example/lib/utils/vector2.dart

```diff
@@ -0,0 +1,17 @@
+class Vector2 {
```

Comment:
Why not use the `Vector2` library that we use in Flame? To not have dependencies or because you need the `const` constructors?

Reply:
To not have dependencies, felt bit overkill to add a whole math lib for just a simple Vector2